src/Bubble/README.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Bubble'
+type: 'component'
+components:
+- Bubble
+status: 'New'
```

Comment:
Might make sense to categorize this component in "Status and metadata"

<img width="172" alt="image" src="https://user-images.githubusercontent.com/2828721/162427864-36b569dd-7906-417d-af12-351865028815.png">

Reply:
Okay, I add